scripts/addPrismaOverrides.ts

```diff
@@ -37,12 +37,12 @@ async function main() {
         rootPkgJson.pnpm.overrides[
           `${pkgJson.name}>@prisma/client`
         ] = `https://registry.npmjs.com/@prisma/client/-/client-${version}.tgz?id=${encodeURIComponent(
-          pkgJson.name,
+          pkgJson.name.replace(/\/|@/g, ''), // remove @ and / before. vitest hiccups otherwise
         )}`;
         rootPkgJson.pnpm.overrides[
           `${pkgJson.name}>prisma`
         ] = `https://registry.npmjs.com/prisma/-/prisma-${version}.tgz?id=${encodeURIComponent(
-          pkgJson.name,
+          pkgJson.name.replace(/\//g, ''),
```

Comment:
Missed `@`?

Reply:
There is no @ in the `prisma` package, only `@prisma/client`

Reply:
But `pkgJson.name` here is the name of the `example/*/package.json` files, not `prisma` nor `@prisma/client` 🤔

Reply:
Oh right - duh... not sure why its working then 🤷‍♂️🤷‍♂️🤷‍♂️ vitest was only complaining about the former i think 🙄🙄